solidity/random-beacon/test/utils/dkg.ts

```diff
@@ -252,3 +267,24 @@ export async function signDkgResult(
 
   return { members, signingMembersIndices, signaturesBytes }
 }
+
+// Creates a members hash that actively participated in dkg
+export function hashDKGMembers(
```

Comment:
I know it's just a utils function, but maybe it would be worth adding a unit test for it? Especially to confirm misbehaved members filtering.

Reply:
Added 78be281